functest/build_test.go

```diff
@@ -158,6 +175,18 @@ func TestBuildLabels(t *testing.T) {
 		}
 	}
 
+	for _, label := range appsodyCommitLabels {
+		if labelsMap[appsodyCommitKey+label] == nil {
+			t.Errorf("Could not find %s%s label in Docker image!", appsodyCommitKey, label)
+		}
+	}
+
+	if labelsMap[appsodyCommitKey+"message"] != commitMessage {
+		t.Errorf("Expected commit message \"%s\" but found \"%s\"", commitMessage, labelsMap[appsodyCommitKey+"message"])
+	}
+
```

Comment:
Just a suggestion: could we also check for the upstream url?

Reply:
Yes, we could check for a lot more labels. However, I didn't want to get carried away in this PR. The commit message was giving me grief so I wanted to make sure that one was working.